portfolio/src/main/webapp/script.js

```diff
@@ -47,17 +47,55 @@ function newCommentBox(comment){
   return commentBox;
 }
 
+function clearChildren(dom) {
+  while(dom.lastElementChild) {
+    dom.removeChild(dom.lastElementChild);
+  }
+}
+
 /**
- * Adds a greeting to the page using Promises
+ * Adds comments to the page using Promises
  */
 function loadComments() {
-  fetch('/data').then(response => response.json()).then((comments) => {
+  const commentContainer = document.getElementById('comment-container');
+  const parameters = location.search;
```

Comment:
What is location.search used for here? Should this be the limit param?

Reply:
location.search here is the entire query string. Should I change the variable "parameters" to "queryString" to make that more readable? Or is it bad practice to use location.search since it passes all the parameters to fetch at once? Here is where I found location.search https://www.w3schools.com/jsref/prop_loc_search.asp

Reply:
It is not obvious to me what parameters are included in location.search that you care about that. I'm guessing it is the limit parameter. I would think you would get it from the selector element. Am I missing something?

Reply:
ohh okay I get it now, thanks!